src/bin/rustfmt.rs

```diff
@@ -119,67 +115,33 @@ fn match_cli_path_or_file(config_path: Option<PathBuf>,
     resolve_config(input_file)
 }
 
-fn update_config(config: &mut Config, matches: &Matches) -> Result<(), String> {
+fn update_config(config: &mut Config, matches: &Matches) -> Result<()> {
     config.verbose = matches.opt_present("verbose");
     config.skip_children = matches.opt_present("skip-children");
 
-    let write_mode = matches.opt_str("write-mode");
-    match matches.opt_str("write-mode").map(|wm| WriteMode::from_str(&wm)) {
```

Comment:
No worries, I can drop it. I kept it as a second commit because it wasn't the same "flavour" as the first. :-)